app/src/main/java/com/concordia/mcga/fragments/NavigationFragment.java

```diff
@@ -327,8 +418,10 @@ private void showBuildingInfoFragment(boolean isVisible) {
     private void showDirectionsFragment(boolean isVisible) {
         if (isVisible) {
             getChildFragmentManager().beginTransaction().show(directionsFragment).commit();
+            outdoors = false;
```

Comment:
So we need to know if we are outdoors. It is possible for the directions bottomsheet to be displayed outdoors if the user has a start and destination